node/test/util/merge_bare.js

```diff
@@ -111,7 +113,7 @@ x=U:C3-2 s=Sa:b;C4-2 s=Sa:a;Bmaster=3;Bfoo=4`,
                 theirCommit: "4",
                 ourCommit: "3",
             },
-            "non-ffmerge with ffwd submodule change": {
+            "non-ffmerge with ffwd submodule change xxxx": {
```

Comment:
What's the xxxx?

Reply:
ahh, marked xxxx for debugging purpose, let me remove it